packages/suite/src/support/messages.ts

```diff
@@ -7363,11 +7363,19 @@ export default defineMessages({
     },
     TR_COINJOIN_PHASE_1_MESSAGE: {
         id: 'TR_COINJOIN_PHASE_2_MESSAGE',
-        defaultMessage: 'Joining coins',
+        defaultMessage: 'Establishing connection',
     },
     TR_COINJOIN_PHASE_2_MESSAGE: {
         id: 'TR_COINJOIN_PHASE_3_MESSAGE',
-        defaultMessage: 'U poor now',
+        defaultMessage: 'Registering inputs',
```

Comment:
TR_COINJOIN_PHASE_2_MESSAGE in id,

the message should be Registering outputs

Reply:
1a1e8b643dcab44d90e9c7ac142ad3ac5762308d